RESTserver/src/main/java/com/softserve/if072/restservice/controller/StoreController.java

```diff
@@ -138,7 +141,8 @@ public void deleteStore(@PathVariable int id, HttpServletResponse response) {
     @GetMapping("/{storeId}/products/{productId}")
     @ResponseBody
     @ResponseStatus(value = HttpStatus.OK)
-    public Product getProductFromStore(@PathVariable int storeId, int productId, HttpServletResponse response) {
+    public Product getProductFromStore(@PathVariable("storeId") Integer storeId, @PathVariable ("productId") Integer productId,
```

Comment:
if your path variable has the same name as method argument, than value of the @PathVariable can be omitted.